samples/AppConfiguration/AppConfiguration/Generated/Models/Error.Serialization.cs

```diff
@@ -27,7 +27,14 @@ internal static Error DeserializeError(JsonElement element)
                     {
                         continue;
                     }
-                    type = property.Value.GetString();
+                    if (property.Value.ValueKind == JsonValueKind.Null)
```

Comment:
This condition is now checked twice, which causes no issues during code execution. However, it would be nice to not have it write it again in this situation. The problem is we have no knowledge that it was handled customly when we go to write the new condition.

If we want to solve this, the best thing I can think of is allowing the new condition to have a flag to change it to use `continue;` in this situation, and no longer write the custom condition above. I haven't looked into how easy that is yet.

Reply:
Fixed it. Instead of sending the nullability state to the property serialization writing, I always set it `false` so the null check isn't written.